login.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Log In-MockUp</title>
+		<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+		<link rel="stylesheet" type="text/css" href="css/grid.css">
+		<link rel="stylesheet" type="text/css" href="css/login.css">
+		<link rel="stylesheet" type="text/css" href="css/common.css">
+	</head>
+	<body>
+	<!-- Section 1 --> 
+		<section class="section1 section">
```

Comment:
Judging by your markup, I'd say that these `.section*` / `.col*`(possibly even `.textContainer*`) classes can probably be substituted with something more suitable, e.g., a single `.section` / `.col` class, with an `:nth-child` qualifier. That way, you don't need to pollute your HTML with extra classes and you can extend the CSS to as many indices as you'd like, while keeping it more manageable in the long run.